mvpa2/algorithms/group_clusterthr.py

```diff
@@ -174,18 +174,40 @@ class GroupClusterThreshold(Learner):
             will vary across features yielding a threshold vector. The number
             of bootstrap samples need to be adequate for a desired probability.
             A ``ValueError`` is raised otherwise.""")
+    # Note that  feature_thresh_prob  is relevant only for "cluster forming" so
+    # will be irrelevant for e.g. metric="max_value"
+    # TODO: address in documentation or some RF to make "Cluster*" implementation
+    # specific (e.g. a subclass?)
 
     chunk_attr = Parameter(
-        'chunks',
+        'chunks', constraints=EnsureStr() | EnsureNone(),
         doc="""Name of the attribute indicating the individual chunks from
             which a single sample each is drawn for averaging into a bootstrap
-            sample.""")
+            sample.  If None, no bootstrapping will be performed, and each
+            sample will be used as a NULL result sample, thus there should be
+            a sufficient number of them""")
```

Comment:
the point is that bootstrapping is just a first step needed only when we have insufficient number of NULL samples.  May be we could even manage to separate that out somehow into a generator which would furnish those samples to the actual logic